x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/saml/SamlRealm.java

```diff
@@ -761,7 +763,15 @@ public void onFileChanged(Path file) {
         }
 
         List<String> getAttribute(SamlAttributes attributes) {
-            return parser.apply(attributes);
+            final List<String> attrValue = parser.apply(attributes);
+            logger.trace(
+                () -> new ParameterizedMessage(
+                    "Parser [{}] generated values [{}]",
+                    name,
```

Comment:
This `name` has a lot of detail in it - see `forSetting` below (line 782), so this message doesn't need much (but needs to make sense with the variety of different "name" options).